modules/shared/src/main/scala/org/tessellation/ext/kryo.scala

```diff
@@ -38,6 +39,14 @@ object kryo {
 
   type KryoRegistrationId[C] = Int Refined C
 
+  implicit class MapRegistrationGreaterEqualId[A](mapa: Map[Class[_], KryoRegistrationId[GreaterEqual[A]]]) {
```

Comment:
Why is it necessary? It seems like it's already covered by the existing `union`

Reply:
otherwise, it throws an error that something cannot be `Greater[300] Or Greater[300]`